mkosi/distributions/opensuse.py

```diff
@@ -45,12 +48,19 @@ def install_packages(cls, state: MkosiState, packages: Sequence[str], apivfs: bo
         if updates_url is not None:
             repos += [Repo("repo-update", f"baseurl={updates_url}", fetch_gpgurls(updates_url))]
```

Comment:
this "fetch_gpgurls" needs to be also under the fallback path

Reply:
Does it need to be? I figure it doesn't hurt to do this for opensuse as well. If any new keys pop up, zypper will download them as needed.

Reply:
This breaks offline builds, and it is unnecessary for zypper, so please move it under the fallback branch